test/functional/basic/find-e2e-specs.js

```diff
@@ -332,6 +332,10 @@ describe('XCUITestDriver - find', function () {
       await B.delay(TEST_PAUSE_DURATION);
     });
     it('should find visible elements', async function () {
+      // This test is failing on 14.2... skipping it for now
```

Comment:
mm.
I met a similar issue in ruby_lib_core. iPhone 12 simulator x iOS 14.2 returned the wrong element coordinate but iPhone 11 x iOS 14.2 one worked well. Potentially such combination stuff could be here...

Reply:
I couldn't reproduce locally. I don't think I have time to be testing different combinations in Azure.